packages/zoe/src/contracts/barterExchange.js

```diff
@@ -13,7 +13,7 @@ import { trade, satisfies } from '../contractSupport';
  * https://agoric.com/documentation/zoe/guide/contracts/barter-exchange.html
  *
  * The Barter Exchange only accepts offers that look like
- * { give: { In: amount }, want: { Out: amount} }
+ * { give: { In: amount }, want: { Out: amount} } // TODO
```

Comment:
Hi @erights , what are these TODOs for?

Reply:
To remind me (now you ;) ) to look for all uses, to make sure those are compat with this being an AmountPattern rather than an Amount. Like one we saw in a recent PR ;)